src/base/base.scss

```diff
@@ -76,10 +76,55 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+a,
+button,
+input,
+textarea,
+select {
```

Comment:
`<summary>` could use the new global focus styles:
https://primer-css-9e1ffwglv-primer.vercel.app/css/utilities/details#using-button-styles-with-the-details-summary-element

![image](https://user-images.githubusercontent.com/1306747/145695363-cb230b68-6f7b-4c64-a100-7a4efca211b8.png)

![image](https://user-images.githubusercontent.com/1306747/145695372-dfc26c02-b78b-402b-858b-bbacdf7a7916.png)

Reply:
This is especially noticeable in Firefox.

[summary](https://primer-css-b32u0b8ly-primer.vercel.app/css/utilities/details#custom-caret)
<img width="986" alt="Screen Shot 2022-01-14 at 1 32 08 PM" src="https://user-images.githubusercontent.com/2313998/149567347-93872450-76c1-47f3-97c8-073d600f985d.png">

<br />

versus

<br />

[button](https://primer-css-b32u0b8ly-primer.vercel.app/css/utilities/details#darkened-fullscreen-click-area)
<img width="997" alt="Screen Shot 2022-01-14 at 1 32 23 PM" src="https://user-images.githubusercontent.com/2313998/149567440-cafc6a5e-442d-467f-8a6c-07ade606fe16.png">

Reply:
Thank you, addressed this directly in the `details` CSS